### PR TITLE
Added a Hotkey to temporarily enable Audio Stretching.

### PR DESCRIPTION
**Pull request by Filoppi**
If bound to the same key as "Disable Emulation Speed Limit" and the machine is capable of running the game at 100% speed, it can be used as a workaround for bug:
https://bugs.dolphin-emu.org/issues/10254

### CODE REVIEW

File: Source/Core/DolphinWX/DolphinWX.vcxproj [hunk 1]
```diff
@@ -13,12 +13,12 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{47411FDB-1BF2-48D0-AB4E-C7C41160F898}</ProjectGuid>
     <ProjectName>Dolphin</ProjectName>
-    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
```
Comment (Helios747) [inline Source/Core/DolphinWX/DolphinWX.vcxproj]: If the visual studio files need updating, split them off into a separate PR.

Comment (Filoppi) [reply Source/Core/DolphinWX/DolphinWX.vcxproj]: That was my mistake, i didn't mean to commit it, i've reverted it (though it is still listed in the changed files).
